### PR TITLE
Update ACL token

### PR DESCRIPTION
**Pull request by chelseakomlo**
Adds CLI command to update an ACL token

If global is not set, it will default to updating the existing token's global value to false, as the flag is a boolean defaulting to false.

### CODE REVIEW

File: command/acl_token_update_test.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	// Create a valid token
+	mockToken := mock.ACLToken()
+	mockToken.Policies = []string{acl.PolicyWrite}
+	mockToken.SetHash()
+	assert.Nil(state.UpsertACLTokens(1000, []*structs.ACLToken{mockToken}))
+
+	// Request to update a new token without providing a valid management token
+	invalidToken := mock.ACLToken()
+	os.Setenv("NOMAD_TOKEN", invalidToken.SecretID)
```
Comment (dadgar) [inline command/acl_token_update_test.go]: Will this not have side effects on other tests?

File: command/acl_token_update_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	code := cmd.Run([]string{"-address=" + url, "-name=bar", mockToken.AccessorID})
+	//code := cmd.Run([]string{"-address=" + url, "-policy=foo", "-type=client"})
```
Comment (dadgar) [inline command/acl_token_update_test.go]: dead code

File: command/acl_token_update_test.go (hunk 1 above)
Comment (dadgar) [reply command/acl_token_update_test.go]: Prefer flags

File: website/source/docs/commands/acl/token-update.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+layout: "docs"
+page_title: "Commands: acl token create"
```
Comment (dadgar) [inline website/source/docs/commands/acl/token-update.html.md.erb]: Update these

File: command/acl_token_update_test.go (hunk 1 above)
Comment (chelseakomlo) [reply command/acl_token_update_test.go]: We also do this in other files unfortunately, I'll update these in another PR.

**Pull request merged**